integration-tests/reactive-messaging-context-propagation/src/main/java/io/quarkus/it/kafka/FlowerConsumers.java

```diff
@@ -0,0 +1,60 @@
+package io.quarkus.it.kafka;
+
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.inject.Inject;
+
+import org.eclipse.microprofile.reactive.messaging.Incoming;
+
+import io.quarkus.logging.Log;
+import io.quarkus.test.vertx.VirtualThreadsAssertions;
+import io.smallrye.common.annotation.NonBlocking;
+import io.smallrye.common.annotation.RunOnVirtualThread;
+import io.smallrye.reactive.messaging.annotations.Blocking;
+import io.vertx.core.Context;
+import io.vertx.core.Vertx;
+
+@ApplicationScoped
+public class FlowerConsumers {
```

Comment:
Did you write these on Valentine's day?

Reply:
Every day is Valentine's day when you feel the love